openmetadata-ui/src/main/resources/ui/src/utils/RouterUtils.ts

```diff
@@ -35,3 +39,13 @@ export const inPageSearchOptions = (pathname: string): Array<string> => {
 
   return strOptions;
 };
+
+export const isPageCentered = (pathname: string): boolean => {
+  return (
+    true ||
```

Comment:
Why we are explicitly writing "Boolean" value here?

Reply:
My bad, missed it. Thanks